pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1221,6 +1221,12 @@ def test_dt64_mul_div_numeric_invalid(self, one, dt64_series, box_with_array):
         with pytest.raises(TypeError, match=msg):
             one / obj
 
+    def test_dt64_series_dataframe(self):
```

Comment:
I don't think this would hit the line of code explained in the issue, it would need to hit a json parsing path